zellij-server/src/panes/grid.rs

```diff
@@ -1109,7 +1109,13 @@ impl Grid {
             Some((self.cursor.x, self.cursor.y))
         }
     }
-
+    /// Clears all buffers with text
+    pub fn clear_screen(&mut self) {
+        self.lines_above = Default::default();
+        self.lines_below = Default::default();
+        self.viewport = Default::default();
```

Comment:
Let's add a conditional here so that we don't do this if `alternate_screen` is `Some`. This happens when eg. programs running in the terminal switch to full screen (eg. `htop` and friends - also Zellij itself). I think in such cases we really don't want to do this. We can add a log if that happens.